storage-initializer/rockcraft.yaml

```
@@ -0,0 +1,72 @@
+# Based on https://github.com/kserve/kserve/blob/v0.10.0/python/storage-initializer.Dockerfile
+name: storage-initializer
+summary: Storage initializer for Kserve deployments
+description: "Kserve storage initializer"
+version: "v0.10.0-22.04-1"
```

suggested per [this discussion](https://github.com/canonical/bundle-kubeflow/issues/747)

```suggestion
version: "0.10.0"
```